docusaurus/docs/reference/config-types/host.v1.md

```diff
@@ -0,0 +1,263 @@
+---
+sidebar_label: host.v1
+sidebar_position: 15
+---
+
+# The `host.v1` Config Type
+
+The `host.v1` configuration type defines how edge routers or tunnelers can make outgoing connections
+for associated services. The config type also allows specifying health checks for the service. An
+endpoint in a `host. v1` config can be specified explicitly or can be configured to use information
+passed through from a tunneler intercepting traffic, allowing it to forward traffic.
+
+The `host. v1` type only allows defining a single host endpoint. If multiple endpoints are required,
+use the [`host. v2`](./host.v2.md) config type instead.
+
+## Endpoint Configuration
+
+For services with defined endpoints, the following three attributes are required:
+
+* `address` - an IP or DNS address to connect to
+* `port` - the network port to connect to
+* `protocol` - tcp or udp
+
+**Examples**
+
+This service is hosted by one application server.
+
+```json
+{
+  "address": "192.168.100.1",
+  "port": 80,
+  "protocol": "tcp"
+}
+```
+
+## Forwarding Configuration
+
+Services that are forwarding traffic from an OpenZiti tunneling proxy may use the following
+properties to indicate what should be forwarded:
+
+* `forwardProtocol` - flag indicating that the protocol of the forwarded connection is to be used.
```

Comment:
imo instead of referring to the _forwarded_ connection, I think it would be more correct to describe the `forward*` fields as "indicating that the (protocol|port|address) of the _intercepted_ connection is to be used.

or perhaps more succinctly: connect to the (protocol|port|address) that the intercepted client was connecting to

Reply:
fixed